test/utility/bc-google-analytics.spec.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-undef */
+
+describe('Baltimore County Google Analytics Utility', () => {
+	describe('isExternalLink', () => {
+		it('should return false, if no element is passed in', () => {
+			const actual = bcpl.utility.googleAnalytics.isExternalLink();
+			expect(actual).toEqual(false);
+		});
+
+		it('should return false, the link element contains a link containing the domain bcpl.info', () => {
```

Comment:
In the three test cases here where `false` is expected, are these exact links part of the specification, or is the requirement something more like "domain ends with bcpl.info"?

Reply:
@ascire updated to account for subdomains, please give another look

Reply:
@martypowell I don't see the fix yet?